syntax = "proto3";

message MsgRequest {
  string method_name = 1;
  optional string params = 2;
  bytes blob=3;
}
 
message MsgResponse {
  optional string returns = 1;
  bytes blob=2;
}